src/base/bittorrent/session.cpp

```diff
@@ -1373,14 +1373,12 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
         settingsPack.set_bool(libt::settings_pack::enable_incoming_utp, true);
         settingsPack.set_bool(libt::settings_pack::enable_outgoing_utp, true);
         break;
-
```

Comment:
The coding guidelines mandates this?

@glassez @sledgehammer999 
Do anyone (besides me) also think an empty line between cases is a good idea?

Reply:
I'll undo these